app/uk/gov/hmrc/gform/addresslookup/AddressLookupController.scala

```diff
@@ -629,8 +629,8 @@ class AddressLookupController(
     fastForward: List[FastForward]
   ): Action[AnyContent] =
     auth.authAndRetrieveForm[SectionSelectorType.Normal](formTemplateId, maybeAccessCode, OperationWithForm.EditForm) {
-      implicit request => implicit l => cache => implicit sse => _ =>
-        mkSyntheticFormModelOptics(formComponentId, cache, maybeAccessCode) {
+      implicit request => implicit l => cache => implicit sse => realFormModelOptics =>
+        mkSyntheticFormModelOptics(formComponentId, cache, maybeAccessCode, realFormModelOptics) {
```

Comment:
Is that for the case when we enter the address manually - to make "error" properties the same as parent component properites?

Reply:
Yes, when a user wants to enter an address manually, it creates a synthetic form component in this function. I think we need to copy errorShortName/errorShortNameStart from the `postcodelookup` component.